packages/block-editor/src/components/block-navigation/list.js

```diff
@@ -44,38 +45,48 @@ export default function BlockNavigationList( {
 	selectedBlockClientId,
 	selectBlock,
 	showNestedBlocks,
+	showMovers,
+	isRootItem = true,
 } ) {
+	const hasMovers = showMovers && blocks.length > 1;
+
 	return (
 		/*
 		 * Disable reason: The `list` ARIA role is redundant but
 		 * Safari+VoiceOver won't announce the list otherwise.
 		 */
 		/* eslint-disable jsx-a11y/no-redundant-roles */
-		<ul className="editor-block-navigation__list block-editor-block-navigation__list" role="list">
+		<ul className="editor-block-navigation__list block-editor-block-navigation__list" role={ isRootItem ? 'tree' : 'group' }>
 			{ map( blocks, ( block ) => {
 				const blockType = getBlockType( block.name );
 				const isSelected = block.clientId === selectedBlockClientId;
+				const blockDisplayName = getBlockDisplayName( blockType, block.attributes );
 
 				return (
-					<li key={ block.clientId }>
-						<div className="editor-block-navigation__item block-editor-block-navigation__item">
+					<li key={ block.clientId } role="treeitem">
+						<div
+							className={ classnames( 'editor-block-navigation__item block-editor-block-navigation__item', {
+								'is-selected': isSelected,
+							} ) }
+						>
 							<Button
-								className={ classnames( 'editor-block-navigation__item-button block-editor-block-navigation__item-button', {
-									'is-selected': isSelected,
```

Comment:
Losing this selected state means after moving a block I can't tell which block I just moved.

Reply:
There is selected state still, it was moved from the button to the wrapping div, so that entire row for the block shows as selected. 

I think the issue is that the block doesn't become selected when clicking a mover.